tools/plugin-toolkit/src/index.ts

```diff
@@ -1,3 +1,62 @@
-// (C) 2019-2021 GoodData Corporation
+// (C) 2021 GoodData Corporation
 
-export { mySdkFunction } from "./code/code";
+import { Command, program } from "commander";
+import * as pkg from "../package.json";
+import { addPluginCmdAction } from "./addPluginCmd";
+import { initCmdAction } from "./initCmd";
+import { usePluginCmdAction } from "./usePluginCmd";
+
+program
+    .version(pkg.version)
+    .name("GoodData Plugin Development Toolkit")
+    .option("--accept-untrusted-ssl", "Allows to run the tool with host, that has untrusted ssl certificate")
+    .option("--hostname <hostname>", "URL of your GoodData host")
+    .option("-c, --config <config>", "path to configuration file");
```

Comment:
Should also probably have the tiger/bear switch? It seems a better place since the host is also here, no?

Reply:
i kept it only in init phase as that's where we act on it (replace bear/tiger backend)

Reply:
OK, that makes sense.